lit_nlp/components/metrics.py

```diff
@@ -166,12 +167,11 @@ def compute(self,
     if not labels or not preds:
       return {}
 
-    if len(labels)<2:
-      logging.warning("Number of selected items needs to be bigger than 2")
-      return {}
-
     mse = sklearn_metrics.mean_squared_error(labels, preds)
-    pearsonr = scipy_stats.pearsonr(labels, preds)[0]
+    if len(labels)<2: # Check if only one point selected.
```

Comment:
nit: our style guide prefers a space before and after the "<" symbol

Reply:
Noted, fixed it in a new commit.